src/domains/global/reducer.ts

```diff
@@ -219,6 +221,29 @@ globalReducer.on(setGlobalChartUnderlayAction, (state, { after, before, masterID
   },
 }))
 
+globalReducer.on(centerAroundHighlightAction, (state) => {
+  if (!state.globalChartUnderlay) {
```

Comment:
Maybe it shouldn't dispatch an action if there is no global underlay. I don't have a strong opinion, however I'd try to avoid branching in reducer if it's possible.

Reply:
So this is just a defensive measure for the future to prevent potential bug, and also a way to make typing clear - because of the type:
```
state.globalChartUnderlay: null | {
 after: number
 ...
```
The action is not dispatched when there is no globalChartUnderlay.

Does it change your opinion?